### PR TITLE
#674: Copy PID, creator and URL elements without a xml:lang attribute to all languages

### PR DESCRIPTION
**Pull request by matthew-morris-cessda**
Language tags are not currently required for these fields in profile. These elements tend to have content that is not language specific.

This closes https://github.com/cessda/cessda.cdc.versions/issues/674

### CODE REVIEW

**Review by markusjt — APPROVED** [review]
Looks good to me. Since this closes issue 674, I was just thinking about the dates but I guess we can generally just assume that dates that don't need language tag will be properly using date atrribute, e.g. ```<collDate date="2019" event="single" ></collDate>``` will work just fine as it is. Meanwhile ```<collDate event="single" >2019</collDate>``` would only appear for English but it's kind of the wrong way to do it anyway and if it's instead something like ```<collDate event="single" >Spring 2019</collDate>```, it's English and should have a language tag.

**Comment by matthew-morris-cessda** [comment]
Good point. This PR doesn't currently cover `collDate`, but I can modify it so that it does.

**Comment by markusjt** [comment]
Thanks, I think that's a good idea even if it's then possible for ```collDate``` free text to be there for all languages even though it actually has e.g. English in it without specifying it's English. Also makes sense since language tag is being changed to always be optional for ```collDate``` in DDI-C profile.

File: src/main/java/eu/cessda/pasc/oci/LanguageExtractor.java [hunk 1]
```diff
@@ -168,4 +169,28 @@ private CMMStudyOfLanguage getCmmStudyOfLanguage(CMMStudy cmmStudy, String lang,
...
+    /**
+     * Merge language specific content with all languages content.
+     *
+     * @param map a map with a list of content for each language
+     * @param lang the language to merge
+     * @return a merged list
+     * @param <T> the type of elements
+     */
+    private static <T> List<T> mergeLanguages(Map<String, List<T>> map, String lang) {
+        // Get language specific PIDs, and then language nonspecific PIDs
```
Comment (markusjt) [inline src/main/java/eu/cessda/pasc/oci/LanguageExtractor.java:181]: Just noticed this mentions PIDs even though it's used for other fields too.

**Review by markusjt — APPROVED** [review]
One comment line could be updated to be generic but I'll approve already since it's very minor. Tested these changes on my dev server by editing xml for FSD1000 and the non-language specific values showed up for both en and fi as expected.

**Pull request merged**